src/cicd/pipelines.ts

```diff
@@ -56,6 +66,26 @@ export interface AddCustomStageProps {
   readonly steps: Step[];
 }
 
+export interface CICDPipelineStackProps extends StackProps {
+  readonly environmentId?: string;
+  readonly pipelineName?: string;
+}
+
+export interface AdditionalPipelineProps {
```

Comment:
@LeonLuttenberger I just rebuilt `CodePipelineProps` for now as Omit isn't supported. Maybe we put a pin in this for now, unless you have other ideas.